backend/src/main/java/com/epam/todo/model/ToDo.java

```diff
@@ -0,0 +1,92 @@
+package com.epam.todo.model;
+
+import org.hibernate.annotations.CreationTimestamp;
+
+import javax.persistence.*;
+import java.util.Date;
+import java.util.Objects;
+
+@Entity
+@Table(name = "toDo")
```

Comment:
Avoid to use camelCase in database names like table name and columns
Use snake_case instead.
Every table name has to be plural to in this case "todos"

Reply:
Done.